sonarlint/sonarlint.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>sonarlint</id>
```

Comment:
I would propose sonarlint-vs2015 (since the SonarLint versions for other IDEs might also be packaged on Chocolatey and maybe there will be also a version for a different VS version in the future)